wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WCheckBoxWebElementTest.java

```diff
@@ -0,0 +1,35 @@
+package com.github.bordertech.wcomponents.test;
+
+import com.github.bordertech.wcomponents.test.components.WCheckBoxFieldUI;
+import com.github.bordertech.wcomponents.test.selenium.ByLabel;
+import com.github.bordertech.wcomponents.test.selenium.MultiBrowserRunner;
+import com.github.bordertech.wcomponents.test.selenium.SeleniumJettyTestCase;
+import com.github.bordertech.wcomponents.test.selenium.driver.SeleniumWComponentsWebDriver;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWCheckBoxWebElement;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MultiBrowserRunner.class)
+public class WCheckBoxWebElementTest extends SeleniumJettyTestCase {
+
+
+	/**
+	 * Constructor...
```

Comment:
I know this is a unit test but this doclet content is a bit sub-optimal even by the dismal standards we usually apply. Please make it at least pay lip-service to relevance!